tests/python/test_multidevice.py

```diff
@@ -224,67 +224,67 @@ def definition(self) -> None:
         mlp_dropout_rng_seed = self.define_scalar(None, dtype=DataType.Int)
         self.input = self.define_tensor(
             shape=[b, s, e],
-            contiguity=[True if b > 1 else None, True, True],
+            contiguity=True,
             dtype=DataType.BFloat16,
         )
         self.layernorm0_weight = self.define_tensor(
             shape=[e],
-            contiguity=[True],
+            contiguity=True,
             dtype=DataType.BFloat16,
         )
         self.layernorm0_bias = self.define_tensor(
             shape=[e],
-            contiguity=[True],
+            contiguity=True,
             dtype=DataType.BFloat16,
         )
         self.mha_linear0_weight = self.define_tensor(
-            shape=[e * 3, e],
-            contiguity=[True, True],
+            shape=[d, e * 3 // d, e],
```

Comment:
out of curiosity, what's the reason for not having these sharded earlier? i.e. change in this PR on API isn't expanding any functionality.

Reply:
That belongs to a different change. I'll rebase. Thanks for catching that!